src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -121,6 +121,10 @@
 #define VU_BUILD_REF_BUGZ "https://bugzilla.redhat.com/show_bug.cgi?id=%s"
 #define VU_BUILD_REF_RHSA "https://access.redhat.com/errata/%s"
 
+// Task message queue size
+#define VU_TASK_MSG_MIN_QUEUE_SIZE 1000
```

Comment:
We should check these limits, because they can't be modified by the user without re-compiling.
Specially the max size, considering that surpassing it doesn't generate a failure or an instability, only an increase in the memory.

Reply:
I considered that 100k would be hard to reach if for some reason the task message queue starts to grow giving enough time to address the issue. 
I did a quick test of the heap for different queue sizes here https://github.com/wazuh/wazuh/issues/13343#issuecomment-1125096130

Reply:
We need to consider the amount of messages, not only their size. Also, the max EPS could be limited and a full queue prevents vulnerability detector from working.

Remember that all the scan by interval tasks generate one message for agent and their interval is around 5 minutes by default.

I'd increase `VU_TASK_MSG_MAX_QUEUE_SIZE` and leave the default value as you defined.